packages/immutadot/src/core/toPath.js

```diff
@@ -102,6 +92,50 @@ const allowingArrays = fn => arg => {
   return fn(toString(arg))
 }
 
+const emptyStringParser = str => str.length === 0 ? [] : null
+
+const quotedBracketNotationParser = map(
```

Comment:
Maybe we could test each parser so i will be easier to work on them individually later ?

Reply:
Good idea but could we do this in a later PR ? The current tests cover all of the code so I think it's OK to merge this as is.

Reply:
I agree with @nlepage. An issue could be filed to make sure it's not forgotten.

Reply:
I opened an [issue](https://github.com/Zenika/immutadot/issues/146) about this topic 👍